Insert more details to API Gateway Instalation section (#15)
Sensedia/api-platform-hybrid: kubernetes/README_en.md

Alter the parameter values of the file ``api-platform-hybrid/api-gateway.yaml`` according to the instructions of the section **Changing Modules Versions and Other Parameters**.

> > NOTE: The property ``apigateway_label`` must be filled in with the desired name for the Gateway Pool, and informed later through the opening of a ticket to ** Support and Operations ** team can complete the creation.

Reviewer: ```suggestion
> NOTE: The property ``apigateway_label`` must be filled in with the desired name for the Gateway Pool and informed later through the opening of a ticket so that the **Support and Operations** team can conclude the creation.
```

kubectl get pods -n MY_HYBRID_ENV
```

> > NOTE: The api-gateway pods will not be available until the next step is completed ([Hybrid Environment Activation](#hybrid-environment-activation)).

Reviewer 2: Sugiro alterar este trecho para o seguinte:

`> NOTE: The *api-gateway* pods will not be healthy until the ([Hybrid Environment Activation](#hybrid-environment-activation)) step is completed.`
